cmd/relay-gossip/client/relayclient_test.go

```diff
@@ -90,7 +90,10 @@ func TestGRPCClient(t *testing.T) {
 	}
 	svc.(mock.MockService).EmitRand(true)
 	cancel()
-	for range ch {
+	select {
```

Comment:
this was draining the channel. since it's at the end, i think we can probably just let it leak by removing entirely.

Reply:
Isn't it an expected behavior though to have one more entry coming from channel ?

Reply:
the current code simply drains the channel with no constraint on how many items come out - just that after cancel the channel does eventually close.
Your change checks that 1 item comes out, but not that the channel closes.

Reply:
Ok I was under the impression that since there was only one "EmitRand" before, only one value should have came out, no ? 
But sounds good, I'll remove the check all together.